Tasks/AzureResourceManagerTemplateDeploymentV3/models/DeployAzureRG.ts

```diff
@@ -0,0 +1,94 @@
+import tl = require("azure-pipelines-task-lib/task");
+import msRestAzure = require('azure-arm-rest-v2/azure-arm-common');
+import { AzureRMEndpoint } from 'azure-arm-rest-v2/azure-arm-endpoint';
+
+export class AzureRGTaskParameters {
+
+    public action: string;
+    public resourceGroupName: string;
+    public location: string;
+    public csmFile: string;
+    public csmParametersFile: string;
+    public templateLocation: string;
+    public csmFileLink: string;
+    public csmParametersFileLink: string;
+    public overrideParameters: string;
+    public outputVariable: string;
+    public subscriptionId: string;
+    public endpointPortalUrl: string;
+    public deploymentName: string;
+    public deploymentMode: string;
+    public credentials: msRestAzure.ApplicationTokenCredentials;
+    public deploymentOutputs: string;
+    public addSpnToEnvironment: boolean;
+    public connectedService: string;
+    public deploymentScope: string;
+    public managementGroupId: string;
+
+    private async getARMCredentials(connectedService: string): Promise<msRestAzure.ApplicationTokenCredentials> {
+        var azureEndpoint = await new AzureRMEndpoint(connectedService).getEndpoint();
+        return azureEndpoint.applicationTokenCredentials;
+    }
+
+    public async getAzureRGTaskParameters() : Promise<AzureRGTaskParameters>
+    {
+        try {
+
+            //Deployment Scope
+            this.deploymentScope = tl.getInput("deploymentScope");
+            if(!this.deploymentScope){
+                this.deploymentScope = "Resource Group";
+            }
+
+            this.connectedService = tl.getInput("ConnectedServiceName", true);
+            var endpointTelemetry = '{"endpointId":"' + this.connectedService + '"}';
```

Comment:
Will this not just log `[object object]`? I think we just need to log connection scope property

Reply:
Ohh this is just id

Reply:
I think we should log the connection scope

Reply:
Added the logs